### PR TITLE
UI : Fix the basic ui issue

### PR DESCRIPTION
**Pull request by Ashish8689**
### Describe your changes :

- Change announcement icon as per list
- Change label color and weight

#
### Type of change :

- [x] Improvement

#
### Frontend Preview (Screenshots) :
image
image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
Frontend: @open-metadata/ui

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/utils/SvgUtils.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ import IconAddReaction from '../assets/svg/add-reaction-emoji.svg';
 import IconAdmin from '../assets/svg/admin.svg';
 import IconAlertBell from '../assets/svg/alert-bell.svg';
 import IconAllActivity from '../assets/svg/all-activity.svg';
+import IconAnnouncementsBasicPrimary from '../assets/svg/announcements-basic-primary.svg';
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/utils/SvgUtils.tsx:28]: @Ashish8689, Don't we have this icon already?

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/utils/SvgUtils.tsx:28]: @Sachin-chaurasiya  yes, we have it in black color and this icon is a primary color.

File: openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.util.tsx [hunk 2]
```diff
@@ -53,9 +53,9 @@ export const getThreadFilterDropdownIcon = (threadType: ThreadType | 'ALL') => {
     case ThreadType.Announcement:
       return (
         <SVGIcons
-          alt="Announcement"
+          alt="Conversation"
```
Comment (Sachin-chaurasiya) [inline openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.util.tsx:56]:
```suggestion
          alt="Announcement"
```

Comment (Ashish8689) [reply openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedList/ActivityFeedList.util.tsx]: @Sachin-chaurasiya  updated!

**Review by ShaileshParmar11 — APPROVED** [review]

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**